src/analytics/navigationTracker/navigationTracker.ts

```diff
@@ -157,24 +181,30 @@ export class NavigationTracker extends BaseTracker {
   }
 
   /**
-   * Determines if the given class name is a header menu bar link.
+   * Determines if the given element is a header link
    *
    * @param {string} element - The HTML link element to get the type of.
-   * @return {boolean} Returns true if the header tag contains this element, false otherwise.
+   * @return {boolean} Returns true if the class name of this element includes "govuk-phase-banner", false otherwise.
```

Comment:
Why this comment changed? Have you merge main branch into your branch?

Reply:
Ah, that was a mistake from the merge conflict that came through, i've corrected it